### PR TITLE
PLT-2757 Reverted New Messages indicator in center channel

### PR DESCRIPTION
**Pull request by hmhealey**

### CODE REVIEW

File: webapp/components/posts_view.jsx [hunk 1]
```diff
@@ -550,18 +542,6 @@ export default class PostsView extends React.Component {
...
                 </div>
-                <div
-                    className='post-list__new-messages-below'
-                    onClick={this.scrollToBottomAnimated}
-                    hidden={!this.state.showUnreadMessageAlert}
-                >
-                    <i className='fa fa-angle-down'></i>
-                    &nbsp;
-                    <FormattedMessage
-                        id='posts_view.newMsg'
```
Comment (enahum) [inline webapp/components/posts_view.jsx]: You still need this in the [locale].json files?

Comment (hmhealey) [inline webapp/components/posts_view.jsx]: Ah right, I forgot about those

Comment (hmhealey) [inline webapp/components/posts_view.jsx]: Never mind, it's still used further up for the old New Messages indicator https://github.com/hmhealey/mattermost-platform/blob/plt2757/webapp/components/posts_view.jsx#L313

**Comment by enahum** [comment]
+1

**Pull request merged**